include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -163,6 +162,26 @@ class DataWriter : public DomainEntity
             void* data,
             const InstanceHandle_t& handle);
 
+    /**
```

Comment:
```suggestion
    /** NOT YET IMPLEMENTED
```
Add this tag to make it consistent with the rest of this document